epic_capybara/cli/bara.py

```diff
@@ -96,6 +102,7 @@ def bara(files, match, unmatch, serve):
             leaf_name = key
 
         fig = figure(x_axis_label=leaf_name, y_axis_label="Entries")
+        fig.xaxis.formatter = PrintfTickFormatter(format="%.1e")
```

Comment:
I don't like this one. All axes are now exponential notation, which is less readable. Can we use this only if the x-range is < 1? What is the issue this resolves (too many zeros?)?

Reply:
The issue was overlapping/crowded labels. Happy for it to be changed.

Reply:
Thanks @veprbl for splitting int from floats. My issues would be resolved if we could use `%g`, which would result in "readable" decimal point notation, and fall back to exponential if the numbers are unreadable.

Reply:
```
root [9] printf("%.1g", 0.00001)
1e-05(int) 5
root [10] printf("%.1g", 0.0001)
0.0001(int) 6
```

Reply:
Didn't see this discussion before. Can we use `%g` unconditionally?

Reply:
I've tried %.02g and it's not always appropriate:
<img width="401" alt="image" src="https://github.com/user-attachments/assets/61809566-5acc-4696-a56e-3f2e04820f6c" />

Reply:
I've pushed a version with both of suggestions by @wdconinc , testing welcome.